src/main/java/hudson/plugins/analysis/collector/AnalysisGraphConfiguration.java

```diff
@@ -30,12 +30,12 @@ public AnalysisGraphConfiguration(final Collection<BuildResultGraph> availableGr
 
     @Override
     protected boolean initializeLocal(final String[] localConfiguration) {
-        if (localConfiguration.length == 1) {
-            if ("0".equals(localConfiguration[0])) {
+        if (localConfiguration.length >= 1) {
```

Comment:
No. It's a required change after upgrading analysis-core version (the last release included two additional parameters so `localConfiguration[0]` was pointing to a wrong one).